docs/DEVELOPER.md

```diff
@@ -7,34 +7,42 @@ Welcome! We're glad that you are interested in helping develop Language Forge.
 
 - [Development Environment Quick Start](#development-environment-quick-start)
   - [Supported Development Environments](#supported-development-environments)
+  - [IDE Developer Experience (DX)](#ide-developer-experience-dx)
   - [Project Setup](#project-setup)
   - [Running the App Locally](#running-the-app-locally)
   - [Mobile device testing on a branch](#mobile-device-testing-on-a-branch)
 - [Tests](#tests)
   - [Running Playwright E2E Tests](#running-playwright-e2e-tests)
-  - [Running Protractor E2E Tests](#running-protractor-e2e-tests)
   - [Running Unit Tests](#running-unit-tests)
 - [Other Commands](#other-commands)
   - [Cleanup](#cleanup)
   - [Running dev](#running-dev)
 - [Debugging](#debugging)
   - [PHP Application Debugging](#php-application-debugging)
-  - [PHP Tests Debugging](#php-tests-debugging)
-  - [E2E Tests Debugging](#e2e-tests-debugging)
+  - [Typescript AngularJS Application Debugging](#typescript-angularjs-application-debugging)
+  - [PHP Test Debugging](#php-test-debugging)
+  - [Playwright E2E Test Debugging](#playwright-e2e-test-debugging)
 - [Style Guides](#style-guides)
-  - [PHP](#php)
-  - [JavaScript](#javascript)
   - [Angular & TypeScript](#angular--typescript)
   </details>
 
 ## Development Environment Quick Start
 
 ### Supported Development Environments
 
-Development of Language Forge is supported using [VS Code](https://code.visualstudio.com/download) on either pure Linux or Windows using WSL ([`wsl --install`](https://learn.microsoft.com/en-us/windows/wsl/install)).
+Development of Language Forge is supported using [VS Code](https://code.visualstudio.com/download) on Linux, MacOS or Windows using WSL ([`wsl --install`](https://learn.microsoft.com/en-us/windows/wsl/install)).
 
 On Windows, the project should be opened with the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) VS Code extension, so that all commands and extensions run on WSL.
 
+### IDE Developer Experience (DX)
+
+While Docker is great way to encapsulate all of the dependencies, build tools and run-time environment of the application, IDEs usually require locally installed extensions, tools and runtimes in order to provide type checking, code hints, and debugging capabilities that make for a better developer experience. To this end, the following locally installed tools/dependencies may be installed for good DX:
+
+- PHP 7.3
+- Composer
+- Node and npm
+- .Net SDK
```

Comment:
.Net? Are you just anticipating merging the repos?

Reply:
The .Net SDK is actually referring to LFMerge development, which while a separate repo, is very much a part of Language Forge development.  It was intentional and non-specific, if only to hint towards a more detailed DX section.